webapp/src/App.tsx

```diff
@@ -18,6 +20,15 @@ function App() {
           {/* Using the same element since it's handling both. Normally you'd probably have a separate one */}
           <Route path=":displayName" element={<CreatedBy />} />
         </Route>
+        <Route path="/newbie" element={<ButtonArsh />} />
+
+        <Route
```

Comment:
Thanks for including a test/demo! Can you add a comment that says something like `TODO: REC-28 remove these examples routes when integrating` just a a note to the other devs so they're not scared to delete it later?